code/datums/status_effects/debuffs.dm

```diff
@@ -974,7 +974,7 @@
 
 		// And roughly 75% of their items will take a smack, too
 		for(var/obj/item/thing in carbon_owner.get_all_gear())
-			if(!QDELETED(thing) && prob(75))
+			if(!QDELETED(thing) && prob(75) && !istype(thing, /obj/item/grenade))
```

Comment:
Separate types that you want to exclude into a type blacklist so that there's things that can be added to it.

Reply:
not sure if how I did it is the optimal way of doing it but its got a blacklist now